sale_order_lot_selection/models/procurement.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, fields, models
```

Comment:
This file it's not necessary now , procurement.order not longer available in 12.0